components/card.css

```diff
@@ -28,7 +28,7 @@
   height: 30px;
   border-radius: 8px;
   border: none;
-  box-shadow:5px 5px 5px var(--fifth-color);
+  box-shadow: 5px 5px 5px var(--fifth-color);
   padding: 0px 20px;
```

Comment:
Maybe you can use rather vw and vh or % instead of px to make it more responsive